modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -738,16 +726,14 @@ FeatureFloodCount::visitElementalNeighbors(const Elem * elem, int current_idx, F
   }
 
   // Loop over all active element neighbors
-  for (std::vector<const Elem *>::const_iterator neighbor_it = all_active_neighbors.begin();
-       neighbor_it != all_active_neighbors.end(); ++neighbor_it)
+  for (const auto neighbor_ptr : all_active_neighbors)
   {
-    const Elem * neighbor = *neighbor_it;
-    processor_id_type my_proc_id = processor_id();
+    auto my_proc_id = processor_id();
 
     // Only recurse on elems this processor can see
-    if (neighbor && neighbor->is_semilocal(my_proc_id))
+    if (neighbor_ptr && neighbor_ptr->is_semilocal(my_proc_id))
```

Comment:
This looks familiar. Didn't I just change that to use a cached lookup? Oh, right that's not merged yet.